### PR TITLE
RHDEVDOCS-5795: Capitalization for 1.9 Repo table

### PR DESCRIPTION
**Pull request by nebelsprachen**
• **Aligned team:** Dev Tools

Version(s):

`gitops-docs-1.9`  

Issue:

[RHDEVDOCS#5795](https://issues.redhat.com/browse/RHDEVDOCS-5795)

Link to docs preview:

* https://71947--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/gitops-argo-cd-properties.adoc [hunk 1]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+|`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
+|`applicationSet` 
+|`applicationSet` controller configuration options.
+| `_<object>_`
+a|* _<image>_ - The container image for the `ApplicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+  * _<version>_ - The tag to use with the `ApplicationSet` container image.
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller `(--kubectl-parallelism-limit flag)`.
```
Comment (skopacz1) [inline modules/gitops-argo-cd-properties.adoc:22]: Just checking, ignore if no change is needed - are the parentheses something that you would include when using this flag, or is it used grammatically to introduce extra context to the description? Point being, if the parentheses are only used here in the description and not in the command, I wouldn't include it in the monospace styling:
```suggestion
  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller (`--kubectl-parallelism-limit` flag).
```

File: modules/gitops-argo-cd-properties.adoc [hunk 2]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+
+|`configManagementPlugins`    |Add a configuration management plugin.| `__<empty>__` |
+|`controller`    |Argo CD Application Controller options.| `__<object>__`
+a|* _<processors.Operation>_ - The number of operation processors.
+  * _<processors.Status>_ - The number of status processors.
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * _<appSync>_ - AppSync is used to control the sync frequency of Argo CD applications
```
Comment (skopacz1) [inline modules/gitops-argo-cd-properties.adoc:30]:
```suggestion
  * _<appSync>_ - AppSync is used to control the sync frequency of Argo CD applications.
```

File: modules/gitops-repo-server-properties.adoc [hunk 3]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
+|`resources` |`__<empty>__` |The container compute resources.
+|`mountSAToken` |`false` |Whether the `serviceAccount` token should be mounted to the repo-server pod.
+|`serviceAccount` |`""` |The name of the `ServiceAccount` to use with the repo-server pod.
+|`verifyTLS` |`false` |Whether to enforce strict TLS checking on all components when communicating with repo server.
+|`autoTLS` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate (one of: OpenShift). Currently only available for OpenShift.
+|`image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
+|`version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
+|`logLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are debug, info, error, and warn.
+|`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are text or json.
+|`execTimeout` | `180` |Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize).
```
Comment (skopacz1) [inline modules/gitops-repo-server-properties.adoc:22]: Per [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e) we should use "for example" instead:
```suggestion
|`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
```

**Review by skopacz1 — COMMENTED** [review]
Left a few comments but otherwise LGTM!

File: modules/gitops-argo-cd-properties.adoc [hunk 4]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+a|* _<image>_ - The container image for the `ApplicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
+  * _<version>_ - The tag to use with the `ApplicationSet` container image.
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<parallelismLimit>_ - The kubectl parallelism limit to set for the controller `(--kubectl-parallelism-limit flag)`.
+
+|`configManagementPlugins`    |Add a configuration management plugin.| `__<empty>__` |
+|`controller`    |Argo CD Application Controller options.| `__<object>__`
+a|* _<processors.Operation>_ - The number of operation processors.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:26]:
```suggestion
a|* _<processors.operation>_ - The number of operation processors.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 5]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * _<processors.Status>_ - The number of status processors.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:27]:
```suggestion
  * _<processors.status>_ - The number of status processors.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 6]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * _<resources>_ - The container compute resources.
+  * _<logLevel>_ - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * _<appSync>_ - AppSync is used to control the sync frequency of Argo CD applications
+  * _<sharding.enabled>_ - Enable sharding on the Argo CD Application Controller component. This property is used to manage a large number of clusters to relieve memory pressure on the controller component.
+  * _<sharding.replicas>_ - The number of replicas that will be used to support sharding of the Argo CD Application Controller.
+  * _<env>_ - Environment to set for the application controller workloads.
+
+|`disableAdmin`    |Disables the built-in admin user.|`false` |
+|`gaTrackingID`    |Use a Google Analytics tracking ID.|`__<empty>__` |
+|`gaAnonymizeusers`    |Enable hashed usernames sent to google analytics.|`false` |
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:37]:
```suggestion
|`gaAnonymizeUsers`    |Enable hashed usernames sent to google analytics.|`false` |
```

File: modules/gitops-argo-cd-properties.adoc [hunk 7]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+|`HA`    |High availablity options.| `__<object>__`
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:38]:
```suggestion
|`ha`    |High availablity options.| `__<object>__`
```

File: modules/gitops-argo-cd-properties.adoc [hunk 8]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+
+|`prometheus` |Prometheus configuration options.|`__<object>__`
+a|* _<enabled>_ - Toggle Prometheus support globally for Argo CD.
+  * _<host>_ - The hostname to use for Ingress or Route resources.
+  * _<ingress>_ - Toggles Ingress for Prometheus.
+  * _<route>_ - Route configuration options.
+  * _<size>_ - The replica count for the Prometheus `StatefulSet`.
+
+|`rbac` |RBAC configuration options.|`__<object>__`
+a|* _<defaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` config map. The name of the default role which Argo CD will fall back to, when authorizing API requests.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:70]:
```suggestion
a|* _<defaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` config map. The name of the default role which Argo CD falls back to when authorizing API requests.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 9]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * _<policy>_ - The `policy.csv` property in the `argocd-rbac-cm` config map. CSV data containing user-defined RBAC policies and role definitions.
+  * _<scopes>_ - The scopes property in the `argocd-rbac-cm` config map. Controls which OIDC scopes to examine during RBAC enforcement (in addition to sub scope).
+
+|`redis` |Redis configuration options.|`__<object>__`
+a|* _<autoTLS>_ - Use the provider to create the Redis server's TLS certificate (one of: OpenShift). Currently only available for {OCP}.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:75]:
```suggestion
a|* _<autotls>_ - Use the provider to create the Redis server's TLS certificate (one of: OpenShift). Currently only available for {OCP}.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 10]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+a|* _<autoscale>_ - Server autoscale configuration options.
+  * _<extraCommandArgs>_ - List of arguments added to the existing arguments set by the Operator.
+  * _<grpc>_ - gRPC configuration options.
+  * _<host>_ - The hostname used for Ingress or Route resources.
+  * _<ingress>_ - Ingress configuration for the Argo CD server component.
+  * _<insecure>_ - Toggles the insecure flag for Argo CD server.
+  * _<resources>_ - The container compute resources.
+  * _<replicas>_ - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `Autoscale` is enabled, `replicas` is ignored.
+  * _<route>_ - Route configuration options.
+  * _<service.Type>_ - The `ServiceType` used for the service resource.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:94]:
```suggestion
  * _<service.type>_ - The `ServiceType` used for the service resource.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 11]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<env>_ - Environment to set for the server workloads.
+
+|`sso` |Single Sign-on options.|`__<object>__`
+a|* _<image>_ - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
+  * _<keycloak>_ - Configuration options for Keycloak SSO provider.
+  * _<dex>_ - Configuration options for Dex SSO provider.
+  * _<provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
+  * _<resources>_ - The container compute resources.
+  * _<verifyTLS>_ - Whether to enforce strict TLS checking when communicating with Keycloak service.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:105]:
```suggestion
  * _<verifyTLS>_ - This defines whether to enforce strict TLS checking when communicating with Keycloak service.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 12]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * _<version>_ - The tag to use with the Keycloak container image.
+|`statusBadgeEnabled` |Enable application status badge.|`true` |
+
+|`tls` |TLS configuration options.|`__<object>__`
+a|* _<ca.ConfigMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
+  * _<ca.SecretName>_ - The name of the secret which contains the CA Certificate and Key.
+  * _<initialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.
+|`userAnonyousEnabled` |Enable anonymous user access.|`true` |
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:113]:
```suggestion
|`usersAnonymousEnabled` |Enable anonymous user access.|`true` |
```

File: modules/gitops-argo-cd-properties.adoc [hunk 13]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+|`version` |The tag to use with the container image for all Argo CD components.|Latest Argo CD version|
+
+|`banner` |Add a UI banner message.|`__<object>__`
+a|* _<banner.Content>_ - The banner message content (required if a banner is displayed).
+  * _<banner.URL.SecretName>_ - The banner message link URL (optional).
```
Comment (eromanova97) [inline modules/gitops-argo-cd-properties.adoc:118]:
```suggestion
  * _<banner.url.SecretName>_ - The banner message link URL (optional). 
```
I did not see this property in the linked list, only `banner.url`: https://github.com/redhat-developer/gitops-operator/blob/master/bundle/manifests/argoproj.io_argocds.yaml#L7328 could you please check if the `secretName` part is still relevant? Thank you!

File: modules/gitops-repo-server-properties.adoc [hunk 14]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
-|`AutoTLS` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate (one of: openshift). Currently only available for OpenShift.
-|`Image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
-|`Version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
-|`LogLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are debug, info, error, and warn.
-|`LogFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are text or json.
-|`ExecTimeout` | `180` |Execution timeout in seconds for rendering tools (e.g. Helm, Kustomize).
-|`Env` | `__<empty>__` |Environment to set for the repository server workloads.
-|`Replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.
+|`resources` |`__<empty>__` |The container compute resources.
+|`mountSAToken` |`false` |Whether the `serviceAccount` token should be mounted to the repo-server pod.
```
Comment (eromanova97) [inline modules/gitops-repo-server-properties.adoc:14]:
```suggestion
|`mountSAToken` |`false` |This describes whether the `serviceAccount` token should be mounted to the repo-server pod.
```

File: modules/gitops-repo-server-properties.adoc [hunk 15]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
+|`serviceAccount` |`""` |The name of the `ServiceAccount` to use with the repo-server pod.
```
Comment (eromanova97) [inline modules/gitops-repo-server-properties.adoc:15]:
```suggestion
|`serviceaccount` |`""` |The name of the `serviceaccount` to use with the repo-server pod.
```

File: modules/gitops-repo-server-properties.adoc [hunk 16]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
+|`verifyTLS` |`false` |Whether to enforce strict TLS checking on all components when communicating with repo server.
```
Comment (eromanova97) [inline modules/gitops-repo-server-properties.adoc:16]:
```suggestion
|`verifyTLS` |`false` |This defines whether to enforce strict TLS checking on all components when communicating with repo server.
```

File: modules/gitops-repo-server-properties.adoc [hunk 17]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
+|`autoTLS` |`""` |Provider to use for setting up TLS the repo-server's gRPC TLS certificate (one of: OpenShift). Currently only available for OpenShift.
+|`image` | `argoproj/argocd` |The container image for Argo CD Repo server. This overrides the `ARGOCD_REPOSERVER_IMAGE` environment variable.
+|`version` | same as `.spec.Version` |The tag to use with the Argo CD Repo server.
+|`logLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are debug, info, error, and warn.
```
Comment (eromanova97) [inline modules/gitops-repo-server-properties.adoc:20]:
```suggestion
|`logLevel` | `info` |The log level used by the Argo CD Repo server. Valid options are `debug`, `info`, `error`, and `warn`.
```

File: modules/gitops-repo-server-properties.adoc [hunk 18]
```diff
@@ -10,18 +10,18 @@ The following properties are available for configuring the Repo server component
...
+|`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are text or json.
```
Comment (eromanova97) [inline modules/gitops-repo-server-properties.adoc:21]:
```suggestion
|`logFormat` | `text` |The log format to be used by the Argo CD Repo server. Valid options are `text` or `json`.
```

**Review by eromanova97 — COMMENTED** [review]
I finished the peer review, the comments are mostly similar to the PR for later versions. Thank you!

File: modules/gitops-argo-cd-properties.adoc [hunk 19]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+a|* `<enabled>` - The toggle to start the notifications-controller.
+  * `<image>` - The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.
+  * `<version>` - The tag to use with the Notifications container image.
+  * `<resources>` - The container compute resources.
+  * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+
+|`repositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.| `__<empty>__` |
+|`initialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.| `__<default_Argo_CD_Known_Hosts>__` |
+|`kustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|`__<empty>__` |
+|`oidCConfig` |The OIDC configuration as an alternative to Dex.|`__<empty>__` |
```
Comment (reginapizza) [inline modules/gitops-argo-cd-properties.adoc:59]: I think you have an extra capitalized C in oidc

File: modules/gitops-argo-cd-properties.adoc [hunk 20]
```diff
@@ -10,100 +10,112 @@ The Argo CD Custom Resource consists of the following properties:
...
+  * `<grpc>` - gRPC configuration options.
+  * `<host>` - The hostname used for Ingress or Route resources.
+  * `<ingress>` - Ingress configuration for the Argo CD server component.
+  * `<insecure>` - Toggles the insecure flag for Argo CD server.
+  * `<resources>` - The container compute resources.
+  * `<replicas>` - The number of replicas for the Argo CD server. Must be greater than or equal to `0`. If `autoscale` is enabled, `replicas` is ignored.
+  * `<route>` - Route configuration options.
+  * `<service.type>` - The `ServiceType` used for the service resource.
+  * `<logLevel>` - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
+  * `_<logFormat>` - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
```
Comment (reginapizza) [inline modules/gitops-argo-cd-properties.adoc:96]: you have an extra "_" in front of <logFormat>

**Pull request merged**